### PR TITLE
Twitter feed & homepage layout

### PR DESCRIPTION
**Pull request by Sam152**

### CODE REVIEW

**Comment by Sam152** [comment]
Current homepage:
screen shot 2015-09-22 at 7 31 05 pm

Ported layout and updated twitter block:
screen shot 2015-09-22 at 7 48 21 pm

File: agov/config/install/system.site.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+uuid: cc42599e-45aa-4f6d-8019-2588955450c7
```
Comment (benjy) [inline agov/config/install/system.site.yml]: 1. I don't think uuid should be in exported config files.
2. Can we setup the site email while we're here?
3. Does this import work if you run through the installer? It never used to in D8

File: build.xml [hunk 2]
```diff
@@ -25,7 +25,7 @@
   <property name="app.password"           value="password"                                            override="true" />
   <!-- Development variables. -->
   <property name="app.dev.modules"        value="config_devel default_content agov_default_content"   override="true" />
-  <property name="app.dev.config_modules" value="agov agov_standard_page"                                  override="true" />
+  <property name="app.dev.config_modules" value="agov agov_standard_page agov_homepage"               override="true" />
```
Comment (benjy) [inline build.xml]: Homepage is a dev module?

**Comment by benjy** [comment]
Rest is looking good.

**Comment by Sam152** [comment]
Issue already exists, config devel now removes the uuid for some reason?

File: build.xml (hunk 2 above)
Comment (Sam152) [inline build.xml]: app.dev.config_modules is a list of modules that get passed to config:export for exporting config.

**Comment by benjy** [comment]
Just wanted to try with a core patch I wrote to resolve the sub-type dependencies in typed-data. The patch still needs work but i'm hoping it works for this use case.

**Pull request merged**